packages/firestore/test/unit/local/local_store.test.ts

```diff
@@ -1126,6 +1126,7 @@ function genericLocalStoreTests(
     );
   });
 
+  /* eslint-disable camelcase */
```

Comment:
Can we just fix it?  array_union => arrayUnion search/replace should work, right?